### ISSUE TITLE
Razer BlackWidow V3 Pro Support

### ISSUE THREAD

**Issue by asithade** [1kc/razer-macos#151]
Hi there! First of all, thank you so much for this. My mouse and extended mat work great!

When I initially installed the app, it seemed to pick up my Razer BlackWidow V3 Pro, however now I don't see it in the settings anymore. Any idea why that is?

**Comment by TheAlmightyZach** [comment]
Girlfriend got me the BlackWidow V3 for Christmas, was frustrated that Synapse doesn't work on Mac. Found this, got excited! It works for the mouse (DeathAdder) and works great! Hoping keyboard support can come soon!

**Comment by redescort87** [comment]
I agree Thank you so much I was very disappointed when I found out that the Razer software was not goin to work for my peripherals. I also would like to have support for the Blackwidow v3 added. The software works wonderfully for my mouse (Lancehead te) but my BlackWidow v3 is not recognized by the software. Please add support for this. Thank you!

**Comment by sdanaher** [comment]
wish I knew how to do this, I just got one also for Xmas and hope it works in this app soon!

**Comment by smithbrendan57** [comment]
Hi! Anything on this?

**Comment by dhobi** [comment]
Looks like openrazer is tracking the progress here: https://github.com/openrazer/openrazer/issues/1279

If you want to help out, head over to openrazer and provide a pcap. 👍

**Comment by o0CloudNine0o** [comment]
Im planning to purchase this. Just wondering if there are any progress to the project?

**Comment by aloisaung** [comment]
anyone know how to use  blackwidow v3 pro synapse for mac?

**Comment by ryan-idea42** [comment]
Just got some pull requests through to support Blackwidow V3 Pro when it's connected via USB. The drivers that are being used to support the keyboard in this project only work through the USB interface. 

Sounds like @1kc is working on a new release, which will have these changes in there. 

When I tested these changes on my keyboard, it worked perfectly! :-D

**Comment by 1kc** [comment]
> Just got some pull requests through to support Blackwidow V3 Pro when it's connected via USB. The drivers that are being used to support the keyboard in this project only work through the USB interface.
> 
> Sounds like @1kc is working on a new release, which will have these changes in there.
> 
> When I tested these changes on my keyboard, it worked perfectly! :-D

The new release is here: https://github.com/1kc/razer-macos/releases/tag/v0.4.4

**Comment by marcel151** [comment]
Hi, just mentioned that Razer-macos for the Blackwidow V3 Pro is not working when connected wirelessly and saw that it is only supported when connected wired. Is there any progress on implementing this? 
Also my Deathadder V2 Pro is not recognized when connected to the Blackwidow's dongle (multi-device).

**Comment by Buslowicz** [comment]
> Hi, just mentioned that Razer-macos for the Blackwidow V3 Pro is not working when connected wirelessly and saw that it is only supported when connected wired. Is there any progress on implementing this? Also my Deathadder V2 Pro is not recognized when connected to the Blackwidow's dongle (multi-device).

I have almost identical issue. I had BlackWidow V3 Pro connected via cable and Basilisk Ultimate connected via dongle - it worked well. Now, I bought a Basilisk V3 Pro, bound it with my BlackWidow dongle for multi device under windows, then connected the dongle to my MacBook Pro - none of the above is detected...
